runtime.md

```diff
@@ -90,18 +94,27 @@ Attempting to start an already started container MUST have no effect on the cont
 `run <container-id>`
 
 This operation MUST generate an error if it is not provided the container ID.
+Attempting to run a container that does not exist MUST generate an error.
 This operation MUST invoke the `create` operation, and if there are no errors, then invoke the `start` operation.
 The implementation MAY immediately execute `start` without any pause after `create` completes.
 For example if the implementation used a temporary process to implement `create`, it need not do this in the implementation of `run`.
 
+### Stop
+`stop <container-id>`
+
+This operation MUST generate an error if it is not provided the container ID.
+Attempting to stop a container that is not running MUST have no effect on the container and MUST generate an error.
+This operation MUST stop all of the processes running within the scope of the container.
+This operation MUST NOT delete any resources associated with the container, except for the PID namespace.
```

Comment:
The concern I have is that if we just split create/start and not stop/delete then we're only doing 1/2 a solution because hooks will still need to be there and it'll look inconsistent to only have them for the tail part of the lifecycle.  I'm ok with this PR taking longer to resolve due to it in luding both. In fact, since I think it makes sense to look at the entire workflow I'd actually prefer it so that we don't have to revisit this PR's decisions when we do get around to the stop/delete discussions.